boot/bootutil/src/ed25519_psa.c

```diff
@@ -187,6 +187,7 @@ int exec_revoke(void)
 }
 #endif /* CONFIG_BOOT_KMU_KEYS_REVOCATION */
 
+#if defined(CONFIG_BOOT_SIGNATURE_USING_KMU)
 void nrf_crypto_keys_housekeeping(void)
```

Comment:
Wait. Is there a code that prevents us from calling `nrf_crypto_keys_housekeeping`?